diff --git a/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts b/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
index 864acb55cb5e..9a0519da7d20 100644
--- a/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
+++ b/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
@@ -122,14 +122,6 @@ rmtfs_mem: memory@fe101000 {
 		};
 	};
 
-	battery: battery {
-		compatible = "simple-battery";
-		charge-full-design-microamp-hours = <8720000>;
-		energy-full-design-microwatt-hours = <33600000>;
-		voltage-min-design-microvolt = <3400000>;
-		voltage-max-design-microvolt = <4480000>;
-	};
-
 	bl_vddpos_5p5: bl-vddpos-regulator {
 		compatible = "regulator-fixed";
 		regulator-name = "bl_vddpos_5p5";
@@ -500,33 +492,6 @@ fsa4480_sbu_mux: endpoint {
 			};
 		};
 	};
-
-	ln8000_charger@51 {
-		compatible = "lionsemi,ln8000";
-		reg = <0x51>;
-
-		status = "okay";
-
-		irq-gpios = <&tlmm 36 0x2002>;
-
-		usb-tcpm = <&pm8150b_typec>;
-
-		ln8000_charger,bat-ovp-threshold = <4560>;
-		ln8000_charger,bat-ovp-alarm-threshold = <4525>;
-		ln8000_charger,bus-ovp-threshold = <13000>;
-		ln8000_charger,bus-ovp-alarm-threshold = <11000>;
-		ln8000_charger,bus-ocp-threshold = <3750>;
-		ln8000_charger,bus-ocp-alarm-threshold = <3500>;
-		ln8000_charger,ntc-alarm-cfg  = <226>;
-
-		ln8000_charger,tdie-prot-disable;
-		ln8000_charger,tbus-mon-disable;
-		ln8000_charger,tbat-mon-disable;
-		ln8000_charger,iin-ocp-disable;
-		ln8000_charger,iin-reg-disable;
-		ln8000_charger,tdie-reg-disable;
-		ln8000_charger,vbat-reg-disable;
-	};
 };
 
 &i2c7 {
@@ -680,12 +645,6 @@ vol_up_n: vol-up-n-state {
 	};
 };
 
-&pm8150b_fg {
-	status = "okay";
-	monitored-battery = <&battery>;
-	//power-supplies = <&ln8000_charger>;
-};
-
 &pm8150b_typec {
 	vdd-pdphy-supply = <&vreg_l2a_3p1>;
 	status = "okay";
@@ -706,7 +665,7 @@ PDO_FIXED_USB_COMM |
 					 PDO_FIXED_DUAL_ROLE |
 					 PDO_FIXED_USB_COMM |
 					 PDO_FIXED_DATA_SWAP)
-					 PDO_VAR(5000, 9000, 3000)>;
+					 PDO_VAR(5000, 12000, 3000)>;
 
 		op-sink-microwatt = <10000000>;
 
